### PR TITLE
issue #183: shorten URL pasted into sheet

### PR DESCRIPTION
**Pull request by rawxfish**
Also gets rid of slack link

Screen Shot 2020-11-22 at 11 32 32 PM

### CODE REVIEW

File: google_api_lib/google_api_client.py [hunk 1]
```diff
@@ -92,15 +92,15 @@ def add_puzzle_and_slack_links_to_sheet(self, puzzle_url, slack_channel_id,
                                             sheet_url):
         req_body = {
             'values': [
-                ['Puzzle link', puzzle_url],
-                ['Slack channel', '%s/app_redirect?channel=%s'
-                                  % (settings.SLACK_BASE_URL, slack_channel_id)]
+                [f'=HYPERLINK("{puzzle_url}", "Puzzle Link")'],
+                #['Slack channel', '%s/app_redirect?channel=%s'
+                #                  % (settings.SLACK_BASE_URL, slack_channel_id)]
```
Comment (akirabaruah) [inline google_api_lib/google_api_client.py:97]: Why are these lines commented out? If we don't need them, we should probably just delete them.

**Review by akirabaruah — COMMENTED** [review]
LGTM overall 👍  a bit curious about the commented lines though

File: google_api_lib/google_api_client.py (hunk 1 above)
Comment (rawxfish) [reply google_api_lib/google_api_client.py:97]: I was just being skittish about deleting code. I'll do it...

**Comment by akirabaruah** [comment]
RIP slack. Did we want to remove all slack references from the codebases or just in this google sheet creation logic?

File: google_api_lib/google_api_client.py (hunk 1 above)
Comment (akirabaruah) [reply google_api_lib/google_api_client.py]: haha same, but I guess we've got git to back us up? 💪 

anyway, thanks for the cleanup!

**Review by akirabaruah — APPROVED** [review]

**Comment by rawxfish** [comment]
Depends on how likely it is that discord becomes worse and we want to switch back to slack in the future... 
@asdfryan wrote all the slack integration last year, any thoughts how easy it would be to de- or re-tangle?

**Comment by asdfryan** [comment]
> Depends on how likely it is that discord becomes worse and we want to switch back to slack in the future...
> @asdfryan wrote all the slack integration last year, any thoughts how easy it would be to de- or re-tangle?

YAGNI. Just delete it all IMO.

**Review by erwa — APPROVED** [review]
LGTM

**Pull request merged**